cmd/govim/internal/types/types.go

```diff
@@ -23,6 +23,9 @@ type Buffer struct {
 	Version  int
 	Listener int
 
+	// Loaded reflects vim's "loaded" flag. See :help bufloaded() for details.
```

Comment:
What does "flag" mean here? 

I think `loaded` represents part of Vim's state with respect to a buffer.

Reply:
updated